src/combinatorics/young_diagram.lean

```diff
@@ -63,6 +63,12 @@ instance : set_like young_diagram (ℕ × ℕ) :=
 @[simp] lemma mem_cells {μ : young_diagram} (c : ℕ × ℕ) :
   c ∈ μ.cells ↔ c ∈ μ := iff.rfl
 
+@[simp] lemma mem_mk (c : ℕ × ℕ) (cells) (is_lower_set) :
+  c ∈ young_diagram.mk cells is_lower_set ↔ c ∈ cells := iff.rfl
+
+instance decidable_mem (μ : young_diagram) : decidable_pred (∈ μ) :=
+show decidable_pred (∈ μ.cells), by apply_instance
```

Comment:
I'm not sure how to do that. Is this some command I run locally?

Reply:
I usually do it as a command, but that's because it's how I learned git. I went ahead and did it:
```
git pull --no-rebase origin master
```
You can also do this from within VS Code (I believe it's this: Source Control -> "..." -> Pull/push -> Pull from). It might prompt you to configure your git for the merge strategy -- this is the function of `--no-rebase` above, which makes it do a standard merge.

Now that I've pushed changes, you'll have to pull if you're going to keep editing things locally.

Reply:
@kmill @jcommelin My local repo no longer runs properly since this change. I'm not sure why but when I try to open it, I get the error
```
info: downloading component 'lean'
error: binary package was not provided for 'darwin'
```
I'm worried I broke something. Can one of you check before we finish this PR?